### PR TITLE
add property maps indexing to collections v2

### PR DESCRIPTION
**Pull request by jillxuu**
Screenshot 2024-07-01 at 2 05 24 PM

backfill:
testnet: 18589
mainnet: 169280

### CODE REVIEW

File: rust/processor/src/db/postgres/migrations/2024-07-01-204112_add_collection_properties_to_collections/down.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ALTER TABLE current_collections_v2
+DROP COLUMN collection_properties;
```
Comment (rtso) [inline rust/processor/src/db/postgres/migrations/2024-07-01-204112_add_collection_properties_to_collections/down.sql:2]: add IF EXISTS

**Comment by rtso** [comment]
Need to also update the insert query and backfill query.

**Comment by jillxuu** [comment]
> Need to also update the insert query and backfill query.

could you elaborate?

File: rust/processor/src/processors/token_v2_processor.rs [hunk 2]
```diff
@@ -354,6 +354,7 @@ fn insert_current_collections_v2_query(
                 table_handle_v1.eq(excluded(table_handle_v1)),
                 token_standard.eq(excluded(token_standard)),
                 last_transaction_version.eq(excluded(last_transaction_version)),
+                collection_properties.eq(excluded(collection_properties)),
```
Comment (rtso) [inline rust/processor/src/processors/token_v2_processor.rs:357]: You also need to update insert_collections_v2_query so that it backfills the new column

Comment (jillxuu) [reply rust/processor/src/processors/token_v2_processor.rs:361]: updated!

**Review by yuunlimm — APPROVED** [review]

**Pull request merged**